Razorpay.php

```diff
@@ -1,10 +1,13 @@
 <?php
 
-// Include Requests
-require_once __DIR__.'/libs/Requests-1.6.1/library/Requests.php';
+// Include Requests only if not already defined
+if (class_exists('Requests') === false)
+{
+    require_once __DIR__.'/libs/Requests-1.6.1/library/Requests.php';
 
-// Register requests autoloader
-Requests::register_autoloader();
+    // Register requests autoloader
+    Requests::register_autoloader();
+}
 
```

Comment:
I like the idea of throwing an exception. Require requests.php if the class is not defined. If it is defined, assume it to be the exact Requests library we want. Call `register_autoloader` assuming that, and if it fails, throw an exception saying "Requests library, found but didn't match". We could also do a check against the Requests lib version. (>=1.6 should be good for us).